### PR TITLE
Details endpoint

### LINKED ISSUE #283: Implement `/details` returning JSON
Call SILO with a details action and return the result as JSON.

### PR DESCRIPTION
**Pull request by fengelniederhammer**
resolves #283

### CODE REVIEW

File: lapis2/src/main/kotlin/org/genspectrum/lapis/controller/LapisController.kt [hunk 1]
```diff
@@ -11,42 +11,52 @@ import io.swagger.v3.oas.annotations.responses.ApiResponse
...
-const val REQUEST_SCHEMA_WITH_GROUP_BY_FIELDS = "SequenceFiltersWithGroupByFields"
-const val RESPONSE_SCHEMA_AGGREGATED = "AggregatedResponse"
+const val AGGREGATED_REQUEST_SCHEMA = "AggregatedPostRequest"
+const val DETAILS_REQUEST_SCHEMA = "DetailsPostRequest"
+const val AGGREGATED_RESPONSE_SCHEMA = "AggregatedResponse"
+const val DETAILS_RESPONSE_SCHEMA = "DetailsResponse"
 
 private const val DEFAULT_MIN_PROPORTION = 0.05
+const val AGGREGATED_GROUP_BY_FIELDS_DESCRIPTION =
+    "The fields to stratify by. If empty, only the overall count is returned"
```
Comment (JonasKellerer) [inline lapis2/src/main/kotlin/org/genspectrum/lapis/controller/LapisController.kt:36]: I had to look up the word stratify but it seems the right one :)

File: lapis2/src/main/kotlin/org/genspectrum/lapis/controller/LapisController.kt [hunk 2]
```diff
@@ -118,6 +128,43 @@ class LapisController(private val siloQueryModel: SiloQueryModel, private val re
...
+    @LapisDetailsResponse
+    fun postDetails(
+        @Parameter(schema = Schema(ref = "#/components/schemas/$DETAILS_REQUEST_SCHEMA"))
+        @RequestBody()
+        request: SequenceFiltersRequestWithFields,
+    ): List<DetailsData> {
+        requestContext.filter = request.sequenceFilters
+
+        return siloQueryModel.getDetails(
+            request.sequenceFilters,
```
Comment (JonasKellerer) [inline lapis2/src/main/kotlin/org/genspectrum/lapis/controller/LapisController.kt:164]: Shouldn't this be `request.sequenceFilters.filterKeys { !nonSequenceFilterFields.contains(it) },` as for aggregrated? It seems that `fields` is also passed as a filter?

Comment (JonasKellerer) [reply lapis2/src/main/kotlin/org/genspectrum/lapis/controller/LapisController.kt:164]: Or is it not necessary above?

**Review by JonasKellerer — APPROVED** [review]

File: lapis2/src/main/kotlin/org/genspectrum/lapis/controller/LapisController.kt (hunk 2 above)
Comment (fengelniederhammer) [reply lapis2/src/main/kotlin/org/genspectrum/lapis/controller/LapisController.kt:164]: Ah thanks, good catch.

**Pull request merged**